tests_metricflow/snapshots/test_predicate_pushdown_rendering.py/SqlQueryPlan/DuckDB/test_metric_time_filter_with_two_targets__plan0_optimized.sql

```diff
@@ -0,0 +1,32 @@
+-- Constrain Output with WHERE
+-- Pass Only Elements: ['bookings', 'listing__country_latest']
+-- Aggregate Measures
+-- Compute Metrics via Expressions
+SELECT
+  listing__country_latest
+  , SUM(bookings) AS bookings
+FROM (
+  -- Join Standard Outputs
+  -- Pass Only Elements: ['bookings', 'listing__country_latest', 'metric_time__day']
+  SELECT
+    subq_13.metric_time__day AS metric_time__day
+    , listings_latest_src_28000.country AS listing__country_latest
+    , subq_13.bookings AS bookings
+  FROM (
+    -- Read Elements From Semantic Model 'bookings_source'
+    -- Metric Time Dimension 'ds'
+    -- Pass Only Elements: ['bookings', 'metric_time__day', 'listing']
+    SELECT
+      DATE_TRUNC('day', ds) AS metric_time__day
+      , listing_id AS listing
+      , 1 AS bookings
+    FROM ***************************.fct_bookings bookings_source_src_28000
+  ) subq_13
+  LEFT OUTER JOIN
+    ***************************.dim_listings_latest listings_latest_src_28000
+  ON
+    subq_13.listing = listings_latest_src_28000.listing_id
+) subq_18
+WHERE metric_time__day = '2024-01-01'
```

Comment:
This doesn't look like it got pushed down - is that expected?

Reply:
Yes, time dimensions are not pushed down yet.